src/app-service.tf

```diff
@@ -0,0 +1,39 @@
+resource "azurerm_resource_group" "example" {
+  name     = format("%s-law", local.project)
+  location = var.location
+}
+
+resource "azurerm_app_service_plan" "app_service_plan" {
```

Comment:
maybe it's better to create an app service module
https://github.com/pagopa/cgn-onboarding-portal-infra/blob/master/code/api.tf#L35